src/control/destination/background/mod.rs

```diff
@@ -51,18 +51,30 @@ type UpdateRx<T> = Receiver<PbUpdate, T>;
 /// provided authority to a set of addresses, and ensures that resolution updates are
 /// propagated to all requesters.
 struct Background<T: HttpService<ResponseBody = RecvBody>> {
+    cfg: QueryCfg,
```

Comment:
nit: What do you think about making this `config: Config`. There isn't really any ambiguity within this file, and we get to avoid _cfg_ (which imo isn't a significant win over _config_, whereas _dst_ is big win over _destination_)

Reply:
That seems reasonable to me.

I added the word `Query` because I thought it was worth indicating that this is used to construct destination service queries. I had initially considered `QueryBuilder`, but that made it sound too much like the builder pattern...